analyses/telomerase-activity-prediction/07_EXTEND-at-threshold.Rmd

```diff
@@ -0,0 +1,207 @@
+         key = "extend_type", 
+         value = "extend_score")
+
+# Get the EXTEND original score median ranks for plot ordering
+cancer_order_df <- extend_df %>%
+  filter(extend_type == "extend_original") %>%
+  select(cancer_group_display, 
+         extend_score) %>%
+  mutate(cancer_order = forcats::fct_reorder(cancer_group_display, extend_score),
+         cancer_order = forcats::fct_relevel(cancer_order, "Other", after = Inf))
+cancer_order <- levels(cancer_order_df$cancer_order)
+
+extend_df$cancer_group_display <- factor(extend_df$cancer_group_display, levels = cancer_order)
+
+# Wrangle FPKM into a data frame with only TERC and TERT expression
+fpkm_df <- read_rds(fpkm_file) %>%
+  rownames_to_column("gene") %>% 
+  # keep only genes we are visualizing
+  filter(gene %in% c("TERC", "TERT")) %>%
+  gather(contains("BS"), key = "SampleID", value = "fpkm") %>%
+  right_join(
+    select(extend_threshold_df, 
+           SampleID,
+           extend_score = extend_threshold), 
+    by = "SampleID"
+  ) %>%
+  mutate(log2_fpkm = log2(fpkm+1))
+```
+
+## Re-analysis 1: Distribution of EXTEND scores
+
+Do trends in the overall distributions of EXTEND scores remain the same if performed only on tumors with high `tumor_fraction` values?
+
+
+```{r}
+ggplot(extend_df) + 
+  aes(x = cancer_group_display, 
+      y = extend_score) + 
+  geom_boxplot(fill = "gray60", 
+               # hide the outliers
+               outlier.size = NULL) + 
+  geom_jitter(width = 0.2, size = 0.5) +
+  facet_wrap(~extend_type, nrow = 2) +
+  theme(axis.text.x = element_text(angle = 30, hjust = 1, size = 6))
+
+```
+
+
+The plot above shows distributions before (top) and after (bottom) filtering, ordered by original EXTEND score median values.
+Previously, we observed from https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/b0fdf6f24fe9c021f22e8b7da348708f68877ee5/content/03.results.md?plain=1#L209:
+
+> We found aggressive tumors such as HGGs (DMGs and other high-grade gliomas) and MB had high `EXTEND` scores (**Figure {@fig:Fig4}D**), while low-grade lesions such as schwannomas, GNGs, DNETs, and other low-grade gliomas had among the lowest scores (**Table S3**).
+
+We see the order is broadly the same, so this means overall EXTEND score trends are largely the same.
+There are some minor differences and items to note:
+
+- First, there are no longer DIPGs for this plot (but these were excluded from the main text figure anyways since it's such a small group)
+- It's notable that pediatric meningiomas are _not_ highly aggressive, but have high scores (both without and with thresholding). 
+Meningiomas had high scores when analyzed previously, so this result is not necessarily different, but we might clarify it further in the manuscript.
+- Similarly, schwannomas are not actually terribly low in either set of distributions, so we may want to rethink this emphasis in the manuscript as well.
+
+
+
+## Re-analysis 2: Relationship to TERT and TERC expression
```

Comment:
I wouldn't make this change, but I think I better understand my own previous confusion in the comment below now – it's a bit odd to plot both the original and high purity scores in re-analysis 1 but not here.